### ISSUE TITLE
end of date variable is 202306

### ISSUE THREAD

**Issue by sehoff** [OpenSourceAP/CrossSection#82]
after downloading the new version of the data and working with it, I noticed that the date variable (yyyymm) ends in 202306. You report an accounting lag of 6(?) month, so how can I look at the difference of 18 month?
I am wondering because using the file "dl_signals_add_crsp.R" , CRSP's yyyymm ends in 202112, which means that there is missing return data for yyymm>202112.  I just want to check whether this creates a problem in the final dataset (signed_predictors_dl_wide.csv), or if it is sufficient to drop all rows that have missing return data, i.e., at least those with date > 202112.

**Comment by chenandrewy** [comment]
I believe the answer to your question is here: https://www.openassetpricing.com/faq/#q-future

**Comment by sehoff** [comment]
yes, perfect! Thanks. To avoid confusion, I would suggest to then just drop all dates that are bigger than the end date of the latest release
